controllers/main.go

```diff
@@ -1998,14 +2015,36 @@ func (controller *MainController) Voting(c web.C, r *http.Request) (string, int)
 
 	t := controller.GetTemplate(c)
 
+	type agenda struct {
+		Agenda chaincfg.ConsensusDeployment
+		Status string
+	}
+	agendaInfos, err := dcrDataAgendas()
+	if err != nil {
+		log.Warnf("unable to retrieve data from dcrdata.org: ", err)
```

Comment:
I would recommend constructing this error message inside dcrDataAgendas(). That way, we can easily include the full, actual URL which was requested

Reply:
I moved the url outside, if that's also ok.